src/components/Restoration/ExtendTimeLimit.vue

```diff
@@ -29,8 +29,9 @@
     <section id="approval-type-section"
       class="section-container"
       :class="{ 'invalid-section': !getApprovalTypeValid }"
+      v-if="approvalType == ApprovalTypes.VIA_COURT_ORDER"
```

Comment:
Section was taking space although approval type was approved via registrar. Having the v-if here instead of in the template would get rid of that.